include/wlr/types/wlr_wl_shell.h

```diff
@@ -106,7 +106,7 @@ void wlr_wl_shell_destroy(struct wlr_wl_shell *wlr_wl_shell);
 
 void wlr_wl_shell_surface_ping(struct wlr_wl_shell_surface *surface);
 void wlr_wl_shell_surface_configure(struct wlr_wl_shell_surface *surface,
-	uint32_t edges, int32_t width, int32_t height);
+	enum wl_shell_surface_resize edges, int32_t width, int32_t height);
```

Comment:
This should be a uint32_t, these are flags iirc. Enum value types cannot have more than one value.

Reply:
I believe all values are enumerated: https://wayland.freedesktop.org/docs/html/apa.html#protocol-spec-wl_shell_surface-enum-resize (but this shouldn't be part of this PR, agreed)

Reply:
You're right, no big deal then.